### ISSUE TITLE
Add existing thermostat

### ISSUE THREAD

**Issue by Dominic-070** [swingerman/ha-dual-smart-thermostat#222]
I have an existing thermostat from Tado and am very satisfied with it. It just can't cool. it would be nice if an existing thermostat could be set for heating then adding a cooling switch for cooling.

**Comment by swingerman** [comment]
Yes, that would be cool. This component can do both without tado.
